packages/runtime/lib/config.js

```diff
@@ -105,8 +105,43 @@ async function _transformConfig (configManager, args) {
     configManager.current.serviceMap.set(service.id, service)
   }
 
+  // If there is no entrypoint, autodetect one
+  if (!config.entrypoint) {
+    // If there is only one service, it becomes the entrypoint
+    if (services.length === 1) {
+      services[0].entrypoint = true
+      config.entrypoint = services[0].id
+      hasValidEntrypoint = true
+    } else {
+      // Search if exactly service uses @platformatic/composer
+      const composers = []
+
+      for (const service of services) {
+        if (!service.config) {
+          continue
+        }
+
+        const config = JSON.parse(await readFile(pathResolve(service.path, service.config), 'utf-8'))
```

Comment:
There is no need to do this. The config is already parsed here. Also, it can be other format than JSON.

Reply:
You are right about the different formats.
But the service config file is not parsed here (the runtime config is). I'll update this accordingly.